repose-aggregator/tests/functional-tests/build.gradle

```diff
@@ -108,16 +108,28 @@ processIntegrationTestResources {
 
 integrationTest {
     useJUnit {
-        def runTest = project.findProperty("run") ?: 'Smoke'
+        def runTest = project.findProperty("run") ?: 'smoke'
         switch(runTest) {
-            case "main":
-                excludeCategories "scaffold.category.Bug", "scaffold.category.Intense"
+            case "all":
+                excludeCategories "scaffold.category.Bug"
+                break
+            case "core":
+                includeCategories "scaffold.category.Core"
+                break
+            case "filters":
+                includeCategories "scaffold.category.Filters"
                 break
             case "intense":
                 includeCategories "scaffold.category.Intense"
                 break
-            case "all":
-                excludeCategories "scaffold.category.Bug"
+            case "recipe":
+                includeCategories "scaffold.category.Recipe"
+                break
+            case "services":
+                includeCategories "scaffold.category.Services"
+                break
+            case "xml":
+                includeCategories "scaffold.category.XmlParsing"
```

Comment:
No case for `bug`?  I get that tests in that category are expected to fail and so should not run in most cases, but not providing a case prevents those tests from ever being run by the Gradle build. It seems like we may as well enable those tests to be run even if just to be able to see what tests are in the `bug` category.